.github/workflows/create-azure-self-hosted-runners.yml

```diff
@@ -134,6 +134,15 @@ jobs:
           | jq --raw-output .token)
         echo "::add-mask::$ACTIONS_RUNNER_TOKEN"
         echo ACTIONS_RUNNER_TOKEN=$ACTIONS_RUNNER_TOKEN >> $GITHUB_ENV
+
+        # The Azure VM type we use has blazing-fast local, temporary storage available as the D:\ drive.
+        # The only downside is that, after dellocation, the contents of this disk (including the Actions Runner),
```

Comment:
That'd make for a fine reason to switch the default to _not_ deallocate by default, don't you think @dennisameling ?

Reply:
Makes sense to me! Once we have the logic in place to automatically remove a runner when a job is finished (as you mentioned [here](https://github.com/git-for-windows/git/discussions/4186#discussioncomment-4576493)), and when we have logic in place that creates those runners on-demand when needed, we shouldn't have to deallocate things, but just live with the few mins of setup time for those jobs.

Reply:
> Makes sense to me! Once we have the logic in place to automatically remove a runner when a job is finished (as you mentioned [here](https://github.com/git-for-windows/git/discussions/4186#discussioncomment-4576493)), and when we have logic in place that creates those runners on-demand when needed, we shouldn't have to deallocate things, but just live with the few mins of setup time for those jobs.

Right. And I could imagine that hosted ARM64-based runners will come at some point, anyway. So those few minutes won't be needed forever.